concepts/0519-goal-codes/README.md

```diff
@@ -0,0 +1,149 @@
+
+>#### goal codes
+>##### `aries.sell`
+>Sell something. Assumes two parties (buyer/seller)
+>##### `aries.sell.goods.consumer`
+>Sell tangible goods of interest to general consumers.
+>##### `aries.sell.services.consumer`
+>Sell services of interest to general consumers.
+>##### `aries.sell.services.enterprise`
+>Sell services of interest to enterprises.
+
+#### In DIDComm-based Protocol Specs
+
+Occasionally, goal codes may have meaning only within the context of a specific protocol. In such cases, it may be appropriate to declare the goal codes directly in a protocol spec. This can be done using a section of the RFC as described above.
+
+More commonly, however, a protocol will *accomplish* one or more goals (e.g., when the protocol is fulfilling a co-protocol interface), or will require a participant to *identify* a goal at one or more points in a protocol flow. In such cases, the goal codes are probably declared external to the protocol. If they can be enumerated, they should still be referenced (hyperlinked to their respective definitions) in the protocol RFC.
+
+#### In Governance Frameworks
+Goal codes can also be (re-)declared in a [machine-readable governance framework](../0430-machine-readable-governance-frameworks/README.md). 
+
+## Reference
+
+#### Known Namespace Aliases
+
+No central registry of namespace aliases is maintained; you need not register with an authority to create a new one. Just pick an alias with good enough uniqueness, and socialize it within your community. For convenience of collision avoidance, however, we maintain a table of aliases that are typically used in global contexts, and welcome PRs from anyone who wants to update it.
+
+alias | used by | URI
+-- | -- | --
+aries | Hyperledger Aries Community | TBD
+
+#### Well-known goal codes
+
+The following goal codes are defined here because they already have demonstrated utility, based on early SSI work in Aries and elsewhere.
+
+##### `aries.vc`
+Participate in some form of VC-based interaction. 
+##### `aries.vc.issue`
+Issue a verifiable credential. 
+##### `aries.vc.verify`
+Verify or validate VC-based assertions.
+##### `aries.rel`
+Create, maintain, or end something that humans would consider a relationship. This should not to be confused with building a DIDComm channel. (Building a DIDComm channel is a low-level procedure, not a high-level goal.)
+##### `aries.rel.build`
+Create a relationship. Carries the meaning implied today by a LinkedIn invitation to connect or a Facebook "Friend" request.
+
+## Unresolved questions
+
+- What parts of the design do you expect to resolve through the
```

Comment:
editorial: these need to be removed if there aren't any unresolved questions

Reply:
Removed.